lib/call-cabal-project-to-nix.nix

```diff
@@ -54,7 +54,7 @@ let
             + "For example use `compiler-nix-name = \"ghc865\";` for ghc 8.6.5") ghc
           else
             if compiler-nix-name != null
-              then pkgs.buildPackages.haskell-nix.compiler."${compiler-nix-name}"
+              then pkgs.haskell-nix.compiler."${compiler-nix-name}"
```

Comment:
Comment above is now wrong.

Reply:
Technically it's not :-) I hope the added in-line comment here clears that up.